x-pack/plugins/security_solution/public/timelines/components/open_timeline/edit_timeline_batch_actions.tsx

```diff
@@ -63,7 +66,9 @@ export const useEditTimelineBatchActions = ({
 
   const getBatchItemsPopoverContent = useCallback(
     (closePopover: () => void) => {
-      const isDisabled = isEmpty(selectedItems);
+      const isItemsSelected = !isEmpty(selectedItems);
+      const hasImmutableItems =
+        !isItemsSelected || selectedItems?.some((item) => item.status === TimelineStatus.immutable);
```

Comment:
```suggestion
        selectedItems?.some((item) => item.status === TimelineStatus.immutable);
```

We do not need `!isItemsSelected` if we just want to know if there is immutable in the selected items

Reply:
and it is confusing with the line 98 where we add back the same logic